CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changed Multicard tests to run standalone ([#118](https://github.com/Lightning-AI/lightning-Habana/pull/118))
```

Comment:
Testing and docs change doesn't need to be listed in the chlog

Reply:
taken care in #119